bin/helpers/constants.js

```diff
@@ -145,6 +147,21 @@ const filesToIgnoreWhileUploading = [
   '.yarn/**',
 ];
 
+const readDirOptions = {
+  cwd: '.',
```

Comment:
Why are we not using cypressFolderPath as CWD? We have the user's who have their cypress.json and CWD different.

Reply:
Will be using that only https://github.com/browserstack/browserstack-cypress-cli/pull/148/files#diff-e4adc3c50732024d070d2df81b167010ada80e39e52edfc8f3f48b0e9b44a501R18 https://github.com/browserstack/browserstack-cypress-cli/pull/148/files#diff-a91b9660eaaccd540cc602da6acb2b2f1bcad2dc63d25de89bd569463295d57eR20 this is just for fallback.